apps-rendering/src/components/LiveBlocks/index.tsx

```diff
@@ -38,7 +38,7 @@ const LiveBlocks: FC<LiveBlocksProps> = ({ blocks, format }) => {
 			{/* Accordion? */}
 			{blocks.map((block) => {
 				const blockFirstPublished = pipe(
-					block.firstPublished,
+					some(block.firstPublished),
 					map(Number),
 					toNullable,
 				);
```

Comment:
I think you might be able to drop this intermediate assignment entirely and pass

```
blockFirstPublished={Number(blockFirstPublished)}
```

directly to the `LiveBlockContainer`?

I think that would also allow flattening this whole block into a single-return arrow function?

```tsx
{blocks.map((block) => (
  <LiveBlockContainer
    key={block.id}
    ...
  >
  ...
  </LiveBlockContainer>
))}
```